src/protagonist/Hydra/Collections/HydraUpdate.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Hydra.Model;
+using Newtonsoft.Json;
+
+namespace Hydra.Collections;
```

Comment:
Only generic `Hydra` classes (for https://www.hydra-cg.com/) should go in Hydra assembly